ui/src/components/FaqEntry.vue

```diff
@@ -1,41 +1,54 @@
 <template>
-    <b-card no-body class="mb-1">
-      <b-card-header header-tag="header" class="p-1" role="tab">
-        <b-button block v-b-toggle="[id]" variant="neutral">{{ title }}
-          <b-icon icon="plus" class="plus" font-scale="1.5"></b-icon>
-          <b-icon icon="dash" class="minus" font-scale="1.5"></b-icon></b-button>
-      </b-card-header>
-      <b-collapse :id="id" accordion="faq-accordion" role="tabpanel">
-        <b-card-body>
-          <b-card-text><slot></slot></b-card-text>
-        </b-card-body>
-      </b-collapse>
-    </b-card>
+  <b-card no-body class="mb-1">
+    <b-card-header header-tag="header" class="p-1" role="tab">
+      <b-button block v-b-toggle="[id]" variant="neutral">{{ title }}
+        <b-icon icon="plus" class="plus" font-scale="1.5"></b-icon>
+        <b-icon icon="dash" class="minus" font-scale="1.5"></b-icon></b-button>
+    </b-card-header>
+    <b-collapse :id="id" accordion="faq-accordion" role="tabpanel">
+      <b-card-body>
+        <b-card-text><slot></slot></b-card-text>
+      </b-card-body>
+    </b-collapse>
+  </b-card>
 </template>
 
 <script type="text/javascript">
-  export default {
-    name: 'FaqEntry',
-    props: {
-        id: String,
-        title: String,
-    },
-  }
+export default {
+  name: 'FaqEntry',
+  props: {
+      id: String,
+      title: String,
+  },
+}
 </script>
 
 <style type="text/css" scoped>
   .card-header .btn {
-    text-align: left;
+    text-align: start;
   }
 
   .card-header .plus, .card-header .minus {
     position: absolute;
     top: .5em;
+  }
+
+  /* LTR specific styles */
```

Comment:
Looks like it is possible to simplify by specifying both `left` and `right` value in all cases (ltr and rtl). See https://developer.mozilla.org/en-US/docs/Web/CSS/position?v=example#types_of_positioning and confirm if it works as expected on most browsers.

> If both left and right are specified, left wins when [direction](https://developer.mozilla.org/en-US/docs/Web/CSS/direction) is ltr (English, horizontal Japanese, etc.) and right wins when [direction](https://developer.mozilla.org/en-US/docs/Web/CSS/direction) is rtl (Persian, Arabic, Hebrew, etc.).

Reply:
Simplified it, works as expected.